docs/REST API/rest-api.md

```diff
@@ -9173,6 +9173,12 @@ Returns information about the specified software. By default, `versions` are sor
       "latest_version": "2.04",
       "icon_url": "https://is1-ssl.mzstatic.com/image/thumb/Purple211/v4/f1/65/1e/a4844ccd-486d-455f-bb31-67336fe46b14/AppIcon-1x_U007emarketing-0-7-0-85-220-0.png/512x512bb.jpg",
       "self_service": true,
+      "automatic_install_policies": [
+        {
+          "policy_id": 345,
+          "policy_name": "[Install software] Logic Pro",
+        } 
+      ], 
```

Comment:
@marko-lisica I think this needs to be in the policy, and not in the software title. This will be consistent with non-vpp software: https://github.com/fleetdm/fleet/pull/22315/files

Also, policy and vpp apps are linked to a team, while software title can be across several teams.

Reply:
> And we should add this info to policy endpoints as well.

@getvictor Which endpoints do you think we need to change?

I think we have all necessary data in team policy endpoints and `/spec/policies` endpoint [here](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/API-for-contributors.md?plain=1#L1523).

Reply:
@marko-lisica Yes, you're right. It seems `name` and `software_title_id` will still work for VPP apps.